### PR TITLE
Debug and release Info.plist files

### PR DESCRIPTION
**Pull request by nickmly**
Separates `Info.plist` into 2 different files for debug and release builds for Detox testing.

### CODE REVIEW

**Comment by timarney** [comment]
@dsamojlenko can you have a look at this one?

We want to have a debug Info.plist so we don't end up committing a file back.  Want to avoid this https://github.com/cds-snc/covid-alert-app#ios-local-development

File: ios/CovidShield/Info-debug.plist [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+<plist version="1.0">
+<dict>
+	<key>BGTaskSchedulerPermittedIdentifiers</key>
+	<array>
+		<string>app.covidshield.exposure-notification</string>
+	</array>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleDisplayName</key>
+	<string>COVID Alert</string>
```
Comment (jeberhardt) [inline ios/CovidShield/Info-debug.plist:12]: Since we are doing DEBUG, can we alter the display name? That way we know which app we are launching.

Comment (timarney) [reply ios/CovidShield/Info-debug.plist:12]: Good idea.  @nickmly ☝️

**Comment by jeberhardt** [comment]
Are we going to apply the same tactic to Android?

**Comment by dsamojlenko** [comment]
So, this just sets the default properties file to be `Info-debug.plist`. You would still need to remember to switch `Info.plist` and `Info-debug.plist` files when running release builds. Would be better if we had a way of automatically switching between the configurations.

Maybe a better approach would be using a build phase script like this: https://sveinhal.github.io/2016/02/23/disable-ats-in-debug-builds/

**Pull request merged**